api/cloudcontroller/ccv2/service_plan_test.go

```diff
@@ -208,4 +208,93 @@ var _ = Describe("Service Plan", func() {
 			})
 		})
 	})
+
+	Describe("UpdateServicePlan", func() {
+		When("the service plan exists", func() {
+			var (
+				expectedRequestBody map[string]interface{}
+				warnings            Warnings
+				err                 error
+				public              bool
```

Comment:
I know that there is no test pollution possibility in the way you have the contexts now, but I'm worried about what happens when someone adds a context inside here in the future and they don't realize they need to initialize `public` and `expectedRequestBody`. Imagine that those values aren't related to their testing goals at all. Their new test may pass when they run it, but they don't realize they've introduced a flakey test.

Maybe you could move the BeforeEach block on line 243 up to this level?